GameData/RealismOverhaul/Engine_Configs/NK9V.cfg

```diff
@@ -197,6 +235,19 @@
 		techTransfer = NK-9:50
 	}
 }
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[NK-21]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
+	{
+		name = NK-21
+		ratedBurnTime = 370 // http://www.astronautix.com/n/n1blockv.html
```

Comment:
The main difference between the NK-19 and NK-21 seems to be with the gimbal/thrust mount. so rated burn time should be the same as the NK-19 here.